frame/elections-phragmen/src/weights.rs

```diff
@@ -329,14 +353,16 @@ impl WeightInfo for () {
 	/// The range of component `v` is `[1, 10000]`.
 	/// The range of component `e` is `[10000, 160000]`.
 	fn election_phragmen(c: u32, v: u32, e: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 773_000
-			.saturating_add(Weight::from_ref_time(81_534_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 51_000
-			.saturating_add(Weight::from_ref_time(4_453_000 as u64).saturating_mul(e as u64))
+		// Minimum execution time: 21_545_085 nanoseconds.
+		Weight::from_ref_time(21_640_807_000 as u64)
+			// Standard Error: 233_654
+			.saturating_add(Weight::from_ref_time(25_222_479 as u64).saturating_mul(v as u64))
+			// Standard Error: 14_994
+			.saturating_add(Weight::from_ref_time(1_034_126 as u64).saturating_mul(e as u64))
 			.saturating_add(RocksDbWeight::get().reads(280 as u64))
 			.saturating_add(RocksDbWeight::get().reads((1 as u64).saturating_mul(c as u64)))
 			.saturating_add(RocksDbWeight::get().reads((1 as u64).saturating_mul(v as u64)))
+			.saturating_add(RocksDbWeight::get().writes(6 as u64))
```

Comment:
what happened here?

Reply:
Probably just forgot to re-run the benches.  
There is this idea to have a CI check that checks if the code in a `frame/*` folder changed and then advises to re-run.  
Using the AST-hashes that @sam0x17 found out about would be better, but more complicated.